apps/api-extractor/src/collector/MessageRouter.ts

```diff
@@ -169,7 +168,7 @@ export class MessageRouter {
         return;  // ignore noise
     }
 
-    const messageText: string = TypeScriptMessageFormatter.format(diagnostic.messageText);
+    const messageText: string = `${diagnostic.messageText};`;
```

Comment:
Same issue with semicolon

Reply:
Done in https://github.com/microsoft/rushstack/pull/1615/commits/e70461520e16cc4031a9a9e26853f97fde4b88bc.